lib/rewards-bamboohr.rb

```diff
@@ -46,6 +42,20 @@ def create_anniversary_bonus
 
     private
 
+    def birthdays
+      @birthdays ||= begin
+        birthdays = bamboohr_fetcher.by_emails(emails) if emails.present?
+        birthdays ||= bamboohr_fetcher.today_birthdays
```

Comment:
@timurvafin what do you think about this decision? But I had to exclude this file for checking the `Lint/UselessAssignment` cop, because it returns the following comment:
> lib/rewards-bamboohr.rb:48:9: W: Lint/UselessAssignment: Useless assignment to variable - birthdays.
       birthdays ||= bamboohr_fetcher.today_birthdays

https://github.com/fs/rewards-bamboohr/pull/8/files#diff-11a0d7906801d9dea0eccb85667ad811R42